eng/pipelines/templates/stages/archetype-js-release.yml

```diff
@@ -189,12 +195,22 @@ stages:
                           node ./eng/tools/versioning/increment.js --artifact-name ${{ artifact.name }} --repo-root .
                         displayName: Increment package version
 
+                      - bash: |
+                          node common/scripts/install-run-rush.js install
+                        displayName: "Install dependencies"
+
+                      - bash: |
+                          npm install -g ./common/tools/dev-tool
+                          npm install ./eng/tools/eng-package-utils
+                          node ./eng/tools/eng-package-utils/update-samples.js ${{ artifact.name }}
+                        displayName: Update samples
+
                       - template: /eng/common/pipelines/templates/steps/create-pull-request.yml
                         parameters:
                           RepoName: azure-sdk-for-js
-                          PRBranchName: increment-package-version-${{ parameters.ServiceDirectory }}-$(Build.BuildId)
-                          CommitMsg: "Increment package version after release of ${{ artifact.name }}"
-                          PRTitle: "Increment version for ${{ parameters.ServiceDirectory }} releases"
+                          PRBranchName: post-release-automation-${{ parameters.ServiceDirectory }}-$(Build.BuildId)
+                          CommitMsg: "Increment package version and update samples after release of ${{ artifact.name }}"
```

Comment:
I'm not sure it matters but if there is a case where there are no sample updates do we want the commit message to still have "and update samples"?

Reply:
I think we can rename it to `post release automated changes`